### PR TITLE
[Bulk] Validation: fixing asp net docs cc6

### PR DESCRIPTION
**Pull request by ccard587**
This PR fixes about 41 aspnetdocs missing alternative text.

### CODE REVIEW

File: aspnet/whitepapers/whats-new-in-aspnet-45-and-visual-studio-2012.md [hunk 1]
```diff
@@ -595,19 +595,19 @@ However, this type of UI help for complex properties has not been available in S
 
 For ASP.NET Web Forms controls, Smart Tasks appear on server tags as a small glyph when the insertion point is inside the element:
 
-![](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image6.png)
+![Screenshot that shows for A S P.NET Web Forms controls, Smart Tasks appear on server tags as a small glyph when the insertion point is inside the element.](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image6.png)
```
Comment (Roudabeh1348) [inline aspnet/whitepapers/whats-new-in-aspnet-45-and-visual-studio-2012.md:598]: Let's spell out dot in A S P. NET as it is in alt text for screen readers.

File: aspnet/whitepapers/whats-new-in-aspnet-45-and-visual-studio-2012.md [hunk 2]
```diff
@@ -595,19 +595,19 @@ However, this type of UI help for complex properties has not been available in S
...
 
 The Smart Task expands when you click the glyph or press CTRL+. (dot), just as in the code editors. It then displays shortcuts that are similar to the Smart Tasks in Design view.
 
-![](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image7.png)
+![Screenshot shows that Smart Task expands when you click the glyph or press C T R L + dot, just as in the code editors. The Smart Task in this illustration shows the GridView Tasks options.](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image7.png)
```
Comment (Roudabeh1348) [inline aspnet/whitepapers/whats-new-in-aspnet-45-and-visual-studio-2012.md:602]: C T R L Plus dot

File: aspnet/whitepapers/whats-new-in-aspnet-45-and-visual-studio-2012.md [hunk 3]
```diff
@@ -642,25 +642,25 @@ In large web pages, it can be a good idea to move individual pieces into user co
 
 To make this easier, when you edit Web Forms pages in Source view, you can now select text in a page, right-click it, and then choose Extract to User Control:
 
-![](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image2.jpg)
+![Screenshot that shows to make editing Web Forms pages in Source View easier, you can now select text in a page, right-click it, and then choose Extract to User Control.](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image2.jpg)
```
Comment (Roudabeh1348) [inline aspnet/whitepapers/whats-new-in-aspnet-45-and-visual-studio-2012.md:645]: screenshot that shows how to make? or screenshot that shows to make?

File: aspnet/whitepapers/whats-new-in-aspnet-45-and-visual-studio-2012.md [hunk 4]
```diff
@@ -753,44 +753,44 @@ You can now add JavaScript files to a central list that will be implicitly inclu
...
 
 The following example shows more complex inheritance:
 
-![](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image26.png)
+![Screenshot that shows an example with more complex inheritance.](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image26.png)
 
 The indentation of a rule is determined by its parent rules. Hierarchical indentation is enabled by default, but you can disable it the Options dialog box (Tools, Options from the menu bar):
 
-![](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image27.png)
+![Screenshot that shows the indentation of a rule, which is determined by its parent rules. Hierarchical indentation is enabled by default, but you can disable it the Options dialog box (Tools, Options from the menu bar).](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image27.png)
```
Comment (Roudabeh1348) [inline aspnet/whitepapers/whats-new-in-aspnet-45-and-visual-studio-2012.md:775]: is there a preposition missing in "but you can disable it the Options dialog box"? I suppose it was copy pasted from the content in which case that should be ok.

File: aspnet/whitepapers/whats-new-in-aspnet-45-and-visual-studio-2012.md [hunk 5]
```diff
@@ -753,44 +753,44 @@ You can now add JavaScript files to a central list that will be implicitly inclu
...
 
-![](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image28.png)
+![Screenshot shows that C S S hacks, and now Visual Studio supports the most widely used ones. This support includes IntelliSense and validation of the star and underscore  property hacks.](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image28.png)
 
 Typical selector hacks are also supported so that hierarchical indentation is maintained even when they are applied. A typical selector hack used to target Internet Explorer 7 is to prepend a selector with *\*:first-child + html*. Using that rule will maintain the hierarchical indentation:
 
-![](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image29.png)
+![Screenshot that shows a typical selector hack used to target Internet Explorer 7 to prepend a selector with slash : first-child plus h t m l. Using this rule maintains the hierarchical indentation.](whats-new-in-aspnet-45-and-visual-studio-2012/_static/image29.png)
```
Comment (Roudabeh1348) [inline aspnet/whitepapers/whats-new-in-aspnet-45-and-visual-studio-2012.md:786]: Let's make H T M L or h t m l for the sake of consistency, I believe you used the H T M L more frequently throughout the article.

**Comment by ccard587** [comment]
Global effort to fix build validation errors

@Rick-Anderson - MSFT NOTE: The validation warnings and suggestions in this PR will be fixed by another author in another PR.

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**